### DISCUSSION TITLE
Help setting up RADI locally for testing.

### DISCUSSION THREAD

**Discussion by GauravKumar9920** [JdeRobot/RoboticsAcademy#1919] [Q&A] [answered]
folliwing up to your answer @pariaspe , I tried building it by running build.sh but I'm getting this error -
"ERROR: failed to solve: executor failed running [/bin/sh -c git clone --recursive https://github.com/PX4/PX4-Autopilot.git -b v1.11.3     && cd /PX4-Autopilot     && DONT_RUN=1 make px4_sitl gazebo]: exit code: 2
[+] Building 0.0s (0/0)                                                        
ERROR: invalid tag "jderobot/robotics-academy:": invalid reference format"
I tried looking it up online but was't able to find a working solution for it can you help me with it....

**Comment by pariaspe** [comment]
Hi @GauravKumar9920 , please give some more information. How did you run it? Can you copy the whole output (not only the error)?

**Comment by GauravKumar9920** [comment]
I ran this - 
gaurav@pop-os:~/RoboticsAcademy/scripts$ ./build.sh
and here's the screen shot of the output- 

![Screenshot-20230204184110-1919x954](https://user-images.githubusercontent.com/84905312/216769582-77d59728-08b9-48ab-b42d-b3206880503f.png)

**Reply by pariaspe** [reply]
You're missing a tag name. Try something like `./build.sh test`

**Comment by GauravKumar9920** [comment]
@pariaspe any comments ?

**Comment by GauravKumar9920** [comment, marked as answer]
Hey,
here's what I tried and didn't work..
at first I opened the RoboticsAcademy in a container via remote-open in conatiner, 
![gnome-shell-screenshot-BUYD01](https://user-images.githubusercontent.com/84905312/216847709-518a758f-5866-4675-be9f-7f9f8939474c.png)

following which I went to find the name of the container running.

![gnome-shell-screenshot-CZSF01](https://user-images.githubusercontent.com/84905312/216847830-7111eb2c-3cee-4b11-b6c1-185a91f7571a.png)

from here I went to terminal and ran `./build.sh gallant_bhabha`
the RADI got built but on checking the files it were the same as orignal and not the one with updated files, what am I doing wrong :,)

![gnome-shell-screenshot-XXP8Z1](https://user-images.githubusercontent.com/84905312/216848452-763df8be-c7a3-418f-8aa0-2eb4325eb5cd.png)

PS: I also tried running the code given below the name(3451...) with ./build.sh 3451....   it too didn't work

**Reply by pariaspe** [reply]
You don't need to open a remote container, neither get its name. The `build.sh` command has to be run in your host machine.
About not getting the updated files. which branch are you in (inside the docker container)? Are you checking out to the desired branch? Make sure to check out during building de image [[1](https://github.com/JdeRobot/RoboticsAcademy/blob/8cbf2438e51f1347926aea9ddce96b1868bad908/scripts/Dockerfile#L53)].

**Reply by GauravKumar9920** [reply]
Thanks @pariaspe I was able to setup the changes by editing the Docker file :)
